solutions/web3-fetching-strategies/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "vercel-example--web3-fetching-strategies",
```

Comment:
```suggestion
  "name": "web3-fetching-strategies",
```

Reply:
Hum i'll probably change the plop templates to fit this!

Thanks! done for this PR